internal/cmd/secret/command_file.go

```diff
@@ -25,6 +25,10 @@ func (c *command) newFileCommand() *cobra.Command {
 	return cmd
 }
 
+func (c *command) addConfigFileFlag(cmd *cobra.Command) {
+	cmd.Flags().String("config-file", "", "Path to the configuration properties file.")
```

Comment:
Does this file have a specific format? YAML, JSON, key=value pairs, etc? We should let the user know.

Reply:
updated!